src/main/java/org/gridsuite/actions/server/entities/FiltersContingencyListEntity.java

```diff
@@ -34,12 +39,15 @@
 
     private String nominalVoltageOperator;
 
+    private List<String> countries;
+
     public FiltersContingencyListEntity(String name, FiltersContingencyListAttributes filtersContingencyListAttributes) {
         this.name = name;
         this.equipmentId = filtersContingencyListAttributes.getEquipmentID();
         this.equipmentName = filtersContingencyListAttributes.getEquipmentName();
         this.equipmentType = filtersContingencyListAttributes.getEquipmentType();
         this.nominalVoltage = filtersContingencyListAttributes.getNominalVoltage();
         this.nominalVoltageOperator = filtersContingencyListAttributes.getNominalVoltageOperator();
+        this.countries = emptyIfNull(filtersContingencyListAttributes.getCountries());
```

Comment:
new Haset(...)
Save data as set to not have duplicated stored elements

Reply:
done